install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -13,567 +14,208 @@ toc::[]
-kind: Endpoints
-metadata:
-  name: glusterfs-cluster <1>
-subsets:
-  - addresses:
-      - ip: 192.168.122.221 <2>
-    ports:
-      - port: 1
-  - addresses:
-      - ip: 192.168.122.222 <2>
-    ports:
-      - port: 1 <3>
-----
-<1> This name must match the service name from step 1.
-<2> The `*ip*` values must be the actual IP addresses of a Gluster server, not
-fully-qualified host names.
-<3> The port number is ignored.
-
-
-. Save the endpoints definition to a file, for example
-*_gluster-endpoints.yaml_*, then create the endpoints:
-+
-[source,bash]
-----
-$ oc create -f gluster-endpoints.yaml
-endpoints "glusterfs-cluster" created
-----
-
-
-. Verify that the endpoints were created:
-+
-[source,bash]
-----
-$ oc get endpoints
-NAME                ENDPOINTS                             AGE
-docker-registry     10.1.0.3:5000                         4h
-glusterfs-cluster   192.168.122.221:1,192.168.122.222:1   11s
-kubernetes          172.16.35.3:8443                      4d
-----
+[[install-example-infra]]
+==== Example: {gluster-native} for OpenShift Logging and Metrics
 
+include::install_config/persistent_storage/topics/glusterfs_example_infra.adoc[]
 
-[[gfs-creating-persistent-volume]]
-=== Creating the Persistent Volume
+[[install-example-full]]
+==== Example: {gluster-native} for Applications, Registry, Logging, and Metrics
 
-[NOTE]
-====
-GlusterFS does not support the 'Recycle' reclaim policy.
-====
-
-. Next, define the PV in an object definition before creating it in
-{product-title}:
-+
-.Persistent Volume Object Definition Using GlusterFS
-
-[source,yaml]
-----
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: gluster-default-volume <1>
-spec:
-  capacity:
-    storage: 2Gi <2>
-  accessModes: <3>
-    - ReadWriteMany
-  glusterfs: <4>
-    endpoints: glusterfs-cluster <5>
-    path: myVol1 <6>
-    readOnly: false
-  persistentVolumeReclaimPolicy: Retain <7>
-----
-<1> The name of the volume. This is how it is identified via
-xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent volume
-claims] or from pods.
-<2> The amount of storage allocated to this volume.
-<3> `accessModes` are used as labels to match a PV and a PVC. They currently
-do not define any form of access control.
-<4> The volume type being used, in this case the *glusterfs*
-plug-in.
-<5> The endpoints name that defines the Gluster cluster
-created in xref:creating-gluster-endpoints[Creating Gluster Endpoints].
-<6> The Gluster volume that will be accessed, as shown in the `gluster volume status`
-command.
-<7> The volume reclaim policy `Retain` indicates that the volume will be
-preserved after the pods accessing it terminates. For GlusterFS, the accepted
-values include `Retain`, and `Delete`.
-
-[NOTE]
-====
-Endpoints are name-spaced. Each project accessing the Gluster volume needs its own endpoints.
-====
+include::install_config/persistent_storage/topics/glusterfs_example_full.adoc[]
```

Comment:
We need to QE test this (that said I [in prior comments]) I think there are simpler ways to do this.